datadog-profiling-replayer/Cargo.toml

```diff
@@ -11,7 +11,7 @@ license.workspace = true
 anyhow = "1.0"
 clap = { version = "4.3.21", features = ["cargo", "color", "derive"] }
 datadog-profiling = { path = "../datadog-profiling" }
-datadog-profiling-protobuf = { path = "../datadog-profiling-protobuf", features = ["prost_impls"] }
+libdd-profiling-protobuf = { path = "../libdd-profiling-protobuf", features = ["prost_impls"] }
```

Comment:
I doesn't have to be versioned since datadog-profiling-replayer is not going to be published but should we version the dependency if it has a version defined? WDYT?